shared/src/main/scala/kinesis4cats/producer/fs2/FS2Producer.scala

```diff
@@ -85,6 +86,17 @@ abstract class FS2Producer[F[_], PutReq, PutRes](implicit
     } yield ()
   }
 
+  /** Put records into the producer's buffer, to be batched and produced at a
+    * defined interval
+    *
+    * @param records
+    *   A [[cats.data.NonEmptyList NonEmptyList]] of
+    *   [[kinesis4cats.producer.Record Records]]
+    * @return
+    *   F of Unit
+    */
+  def putN(records: NonEmptyList[Record]): F[Unit] = records.traverse_(put)
```

Comment:
**issue:** I think having a traverse here doesn't really help much, I would've thought this would do an `offer` to the underlying queue in a batched way, e.g. if the queue was `Queue[F, Chunk[Record]]` (which you can stream from using `Stream.fromQueueUnterminatedChunk`).

Reply:
Well, I'm using a `Channel` here, so as to take advantage of the graceful shutdown semantics. `Channel` does have a `sendAll`, but that _closes the Channel_ so I wouldn't want to use that.

Reply:
See https://github.com/typelevel/fs2/blob/main/core/shared/src/main/scala/fs2/concurrent/Channel.scala#L33-L37